From 3cc4f59637a0191ac73b18b1cf9f4180008ca435 Mon Sep 17 00:00:00 2001
From: fundamental <mark.d.mccurry@gmail.com>
Date: Sun, 11 Nov 2012 09:44:32 -0500
Subject: [PATCH 0042/1936] FLTK: squelching 1.1.x warning messages

Some 1.3.x specific fluid flags were in the fl files resulting in a large number
of warning being generated by fluid.
This patch fixes these errors without appearing to modify the presentation of
the compiled UI.
---
 src/UI/ADnoteUI.fl    |  97 +++++++++++++--------------------
 src/UI/BankUI.fl      |  63 ++++++++++------------
 src/UI/EffUI.fl       |  53 ++++++++----------
 src/UI/EnvelopeUI.fl  |  62 ++++++++++------------
 src/UI/FilterUI.fl    |  58 +++++++++-----------
 src/UI/LFOUI.fl       |  35 ++++++------
 src/UI/MasterUI.fl    | 121 +++++++++++++++++-------------------------
 src/UI/OscilGenUI.fl  |  97 +++++++++++++--------------------
 src/UI/PADnoteUI.fl   |  62 ++++++++++------------
 src/UI/PartUI.fl      | 103 ++++++++++++++---------------------
 src/UI/ResonanceUI.fl |  57 ++++++++------------
 src/UI/SUBnoteUI.fl   |  42 +++++++--------
 src/UI/VirKeyboard.fl |  89 ++++++++++++-------------------
 13 files changed, 384 insertions(+), 555 deletions(-)

diff --git a/src/UI/ADnoteUI.fl b/src/UI/ADnoteUI.fl
index e7105740..ed20e82e 100644
--- a/src/UI/ADnoteUI.fl
+++ b/src/UI/ADnoteUI.fl
@@ -1,59 +1,53 @@
 # data file for the Fltk User Interface Designer (fluid)
-version 1.0300 
+version 1.0110 
 header_name {.h} 
 code_name {.cc}
-decl {//Copyright (c) 2002-2005 Nasca Octavian Paul} {private local
-} 
+decl {//Copyright (c) 2002-2005 Nasca Octavian Paul} {} 
 
-decl {//License: GNU GPL version 2 or later} {private local
-} 
+decl {//License: GNU GPL version 2 or later} {} 
 
-decl {\#include "../Params/ADnoteParameters.h"} {public local
+decl {\#include "../Params/ADnoteParameters.h"} {public
 } 
 
-decl {\#include "../Misc/Util.h"} {public local
+decl {\#include "../Misc/Util.h"} {public
 } 
 
-decl {\#include "../Misc/Master.h"} {public local
+decl {\#include "../Misc/Master.h"} {public
 } 
 
-decl {\#include "ResonanceUI.h"} {public local
+decl {\#include "ResonanceUI.h"} {public
 } 
 
-decl {\#include <FL/Fl_Box.H>} {public local
+decl {\#include <FL/Fl_Box.H>} {public
 } 
 
-decl {\#include <FL/Fl_Group.H>} {public local
+decl {\#include <FL/Fl_Group.H>} {public
 } 
 
-decl {\#include <math.h>} {private local
-} 
+decl {\#include <math.h>} {} 
 
-decl {\#include <stdio.h>} {private local
-} 
+decl {\#include <stdio.h>} {} 
 
-decl {\#include <stdlib.h>} {private local
-} 
+decl {\#include <stdlib.h>} {} 
 
-decl {\#include <string.h>} {private local
-} 
+decl {\#include <string.h>} {} 
 
-decl {\#include "WidgetPDial.h"} {public local
+decl {\#include "WidgetPDial.h"} {public
 } 
 
-decl {\#include "EnvelopeUI.h"} {public local
+decl {\#include "EnvelopeUI.h"} {public
 } 
 
-decl {\#include "LFOUI.h"} {public local
+decl {\#include "LFOUI.h"} {public
 } 
 
-decl {\#include "FilterUI.h"} {public local
+decl {\#include "FilterUI.h"} {public
 } 
 
-decl {\#include "OscilGenUI.h"} {public local
+decl {\#include "OscilGenUI.h"} {public
 } 
 
-decl {\#include "PresetsUI.h"} {public local
+decl {\#include "PresetsUI.h"} {public
 } 
 
 class ADvoicelistitem {open : {public Fl_Group}
@@ -61,7 +55,7 @@ class ADvoicelistitem {open : {public Fl_Group}
   Function {make_window()} {open private
   } {
     Fl_Window ADnoteVoiceListItem {open
-      private xywh {259 713 615 100} type Double box UP_FRAME
+      private xywh {262 736 615 100} type Double box UP_FRAME
       class Fl_Group visible
     } {
       Fl_Group voicelistitemgroup {
@@ -172,14 +166,10 @@ ADnoteVoiceListItem->redraw();} {}
     code {ADnoteVoiceListItem->hide();
 //delete(ADnoteVoiceListItem);} {}
   }
-  decl {ADnoteParameters *pars;} {private local
-  }
-  decl {int nvoice;} {private local
-  }
-  decl {Oscilloscope *osc;} {private local
-  }
-  decl {Master *master;} {private local
-  }
+  decl {ADnoteParameters *pars;} {}
+  decl {int nvoice;} {}
+  decl {Oscilloscope *osc;} {}
+  decl {Master *master;} {}
 } 
 
 class ADvoiceUI {open : {public Fl_Group}
@@ -188,7 +178,7 @@ class ADvoiceUI {open : {public Fl_Group}
   } {
     Fl_Window ADnoteVoiceParameters {
       label Voice open
-      xywh {81 282 765 590} type Double box NO_BOX
+      xywh {84 305 765 590} type Double box NO_BOX
       class Fl_Group visible
     } {
       Fl_Group voiceparametersgroup {open
@@ -808,18 +798,12 @@ if (oscedit!=NULL) {
 };
 //delete (ADnoteVoiceParameters);} {}
   }
-  decl {int nvoice;} {private local
-  }
-  decl {ADnoteParameters *pars;} {private local
-  }
-  decl {OscilEditor *oscedit;} {private local
-  }
-  decl {Oscilloscope *osc;} {private local
-  }
-  decl {Oscilloscope *oscFM;} {private local
-  }
-  decl {Master *master;} {private local
-  }
+  decl {int nvoice;} {}
+  decl {ADnoteParameters *pars;} {}
+  decl {OscilEditor *oscedit;} {}
+  decl {Oscilloscope *osc;} {}
+  decl {Oscilloscope *oscFM;} {}
+  decl {Master *master;} {}
 } 
 
 class ADnoteUI {open : {public PresetsUI_}
@@ -828,7 +812,7 @@ class ADnoteUI {open : {public PresetsUI_}
   } {
     Fl_Window ADnoteGlobalParameters {
       label {ADsynth Global Parameters of the Instrument} open
-      xywh {454 296 540 430} type Double visible
+      xywh {457 319 540 430} type Double visible
     } {
       Fl_Group {} {
         label FREQUENCY open
@@ -1037,7 +1021,7 @@ resui->resonancewindow->show();}
     }
     Fl_Window ADnoteVoice {
       label {ADsynth Voice Parameters} open
-      xywh {907 477 765 620} type Double visible
+      xywh {512 361 765 620} type Double visible
     } {
       Fl_Group advoice {
         xywh {0 0 765 585}
@@ -1170,14 +1154,9 @@ for (int i=0;i<NUM_VOICES;i++) voicelistitem[i]->refreshlist();
 resui->refresh();
 currentvoicecounter->do_callback();} {}
   }
-  decl {ADnoteParameters *pars;} {private local
-  }
-  decl {ResonanceUI *resui;} {private local
-  }
-  decl {Master *master;} {private local
-  }
-  decl {int nvoice;} {private local
-  }
-  decl {ADvoicelistitem *voicelistitem[NUM_VOICES];} {private local
-  }
+  decl {ADnoteParameters *pars;} {}
+  decl {ResonanceUI *resui;} {}
+  decl {Master *master;} {}
+  decl {int nvoice;} {}
+  decl {ADvoicelistitem *voicelistitem[NUM_VOICES];} {}
 } 
diff --git a/src/UI/BankUI.fl b/src/UI/BankUI.fl
index 13c5eade..70c273a2 100644
--- a/src/UI/BankUI.fl
+++ b/src/UI/BankUI.fl
@@ -1,41 +1,39 @@
 # data file for the Fltk User Interface Designer (fluid)
-version 1.0300 
+version 1.0110 
 header_name {.h} 
 code_name {.cc}
-decl {//Copyright (c) 2002-2005 Nasca Octavian Paul} {private local
-} 
+decl {//Copyright (c) 2002-2005 Nasca Octavian Paul} {} 
 
-decl {//License: GNU GPL version 2 or later} {private local
-} 
+decl {//License: GNU GPL version 2 or later} {} 
 
-decl {\#include <stdlib.h>} {public local
+decl {\#include <stdlib.h>} {public
 } 
 
-decl {\#include <stdio.h>} {public local
+decl {\#include <stdio.h>} {public
 } 
 
-decl {\#include <string.h>} {public local
+decl {\#include <string.h>} {public
 } 
 
-decl {\#include <FL/Fl_Button.H>} {public local
+decl {\#include <FL/Fl_Button.H>} {public
 } 
 
-decl {\#include <FL/Fl_File_Chooser.H>} {public local
+decl {\#include <FL/Fl_File_Chooser.H>} {public
 } 
 
-decl {\#include "../Misc/Master.h"} {public local
+decl {\#include "../Misc/Master.h"} {public
 } 
 
-decl {\#include "../Misc/Part.h"} {public local
+decl {\#include "../Misc/Part.h"} {public
 } 
 
-decl {\#include "../Misc/Bank.h"} {public local
+decl {\#include "../Misc/Bank.h"} {public
 } 
 
-decl {\#include "../Misc/Config.h"} {public local
+decl {\#include "../Misc/Config.h"} {public
 } 
 
-decl {\#include "../Misc/Util.h"} {public local
+decl {\#include "../Misc/Util.h"} {public
 } 
 
 class BankProcess_ {} {
@@ -43,7 +41,7 @@ class BankProcess_ {} {
   } {
     code {;} {}
   }
-  decl {Bank *bank;} {public local
+  decl {Bank *bank;} {public
   }
 } 
 
@@ -54,7 +52,8 @@ class BankSlot {open : {public Fl_Button,BankProcess_}
     code {what=NULL;
 whatslot=NULL;
 nslot=0;
-nselected=NULL;} {}
+nselected=NULL;} {selected
+    }
   }
   Function {handle(int event)} {return_type int
   } {
@@ -101,12 +100,9 @@ if (*nselected==nslot)
 
 copy_label(bank->getnamenumbered(nslot).c_str());} {}
   }
-  decl {int *what,*whatslot,nslot,highlight, *nselected;} {private local
-  }
-  decl {void (BankProcess_:: *fnc)(void);} {private local
-  }
-  decl {BankProcess_ *bp;} {private local
-  }
+  decl {int *what,*whatslot,nslot,highlight, *nselected;} {}
+  decl {void (BankProcess_:: *fnc)(void);} {}
+  decl {BankProcess_ *bp;} {}
 } 
 
 class BankUI {open : {public BankProcess_}
@@ -114,8 +110,8 @@ class BankUI {open : {public BankProcess_}
   Function {make_window()} {open
   } {
     Fl_Window bankuiwindow {
-      label Bank selected
-      xywh {807 447 785 575} type Double
+      label Bank
+      xywh {492 406 785 575} type Double
       code0 {o->label(bank->bankfiletitle.c_str());}
       code1 {if (bank->bankfiletitle.empty()) o->label ("Choose a bank from the bank list on the left (or go to settings if to configure the bank location) or choose 'New Bank...' to make a new bank.");} visible
     } {
@@ -362,16 +358,11 @@ removeselection();
 if (beginnerui) modeselect->hide();
 	else modeselect->show();} {}
   }
-  decl {BankSlot *bs[BANK_SIZE];} {private local
-  }
-  decl {int slot,what;//"what"=what button is pressed} {private local
-  }
-  decl {int mode,*npart,nselected;} {private local
-  }
-  decl {Master *master;} {private local
-  }
-  decl {void (BankProcess_::* fnc)(void);} {private local
-  }
-  decl {Fl_Valuator *cbwig;} {public local
+  decl {BankSlot *bs[BANK_SIZE];} {}
+  decl {int slot,what;//"what"=what button is pressed} {}
+  decl {int mode,*npart,nselected;} {}
+  decl {Master *master;} {}
+  decl {void (BankProcess_::* fnc)(void);} {}
+  decl {Fl_Valuator *cbwig;} {public
   }
 } 
diff --git a/src/UI/EffUI.fl b/src/UI/EffUI.fl
index 762aa4d0..07e32678 100644
--- a/src/UI/EffUI.fl
+++ b/src/UI/EffUI.fl
@@ -1,44 +1,42 @@
 # data file for the Fltk User Interface Designer (fluid)
-version 1.0300 
+version 1.0110 
 header_name {.h} 
 code_name {.cc}
-decl {//Copyright (c) 2002-2005 Nasca Octavian Paul} {private local
-} 
+decl {//Copyright (c) 2002-2005 Nasca Octavian Paul} {} 
 
-decl {//License: GNU GPL version 2 or later} {private local
-} 
+decl {//License: GNU GPL version 2 or later} {} 
 
-decl {\#include <stdlib.h>} {public local
+decl {\#include <stdlib.h>} {public
 } 
 
-decl {\#include <stdio.h>} {public local
+decl {\#include <stdio.h>} {public
 } 
 
-decl {\#include <string.h>} {public local
+decl {\#include <string.h>} {public
 } 
 
-decl {\#include "../globals.h"} {public local
+decl {\#include "../globals.h"} {public
 } 
 
-decl {\#include "WidgetPDial.h"} {public local
+decl {\#include "WidgetPDial.h"} {public
 } 
 
-decl {\#include "EnvelopeUI.h"} {public local
+decl {\#include "EnvelopeUI.h"} {public
 } 
 
-decl {\#include "FilterUI.h"} {public local
+decl {\#include "FilterUI.h"} {public
 } 
 
-decl {\#include "../Misc/Util.h"} {public local
+decl {\#include "../Misc/Util.h"} {public
 } 
 
-decl {\#include "../Effects/EffectMgr.h"} {public local
+decl {\#include "../Effects/EffectMgr.h"} {public
 } 
 
-decl {\#include "PresetsUI.h"} {public local
+decl {\#include "PresetsUI.h"} {public
 } 
 
-decl {\#include "common.H"} {public local
+decl {\#include "common.H"} {public
 } 
 
 class EQGraph {: {public Fl_Box}
@@ -158,14 +156,11 @@ return(20.0*pow((float)1000.0,x));} {}
     code {if (freq<0.00001) freq=0.00001;
 return(log(freq/20.0)/log(1000.0));} {}
   }
-  decl {int oldx,oldy;} {private local
-  }
-  decl {float khzval;} {public local
-  }
-  decl {EffectMgr *eff;} {private local
-  }
-  decl {int maxdB;} {private local
+  decl {int oldx,oldy;} {}
+  decl {float khzval;} {public
   }
+  decl {EffectMgr *eff;} {}
+  decl {int maxdB;} {}
 } 
 
 class EffUI {open : {public Fl_Group,public PresetsUI_}
@@ -1473,10 +1468,8 @@ this->show();} {}
   } {
     code {refresh(eff);} {}
   }
-  decl {EffectMgr *eff;} {private local
-  }
-  decl {int eqband;} {private local
-  }
+  decl {EffectMgr *eff;} {}
+  decl {int eqband;} {}
 } 
 
 class SimpleEffUI {open : {public Fl_Group,public PresetsUI_}
@@ -2354,8 +2347,6 @@ this->show();} {}
   Function {refresh()} {} {
     code {refresh(eff);} {}
   }
-  decl {EffectMgr *eff;} {private local
-  }
-  decl {int eqband;} {private local
-  }
+  decl {EffectMgr *eff;} {}
+  decl {int eqband;} {}
 } 
diff --git a/src/UI/EnvelopeUI.fl b/src/UI/EnvelopeUI.fl
index d6281bad..34a41c67 100644
--- a/src/UI/EnvelopeUI.fl
+++ b/src/UI/EnvelopeUI.fl
@@ -1,44 +1,42 @@
 # data file for the Fltk User Interface Designer (fluid)
-version 1.0300 
+version 1.0110 
 header_name {.h} 
 code_name {.cc}
-decl {//Copyright (c) 2002-2005 Nasca Octavian Paul} {private local
-} 
+decl {//Copyright (c) 2002-2005 Nasca Octavian Paul} {} 
 
-decl {//License: GNU GPL version 2 or later} {private local
-} 
+decl {//License: GNU GPL version 2 or later} {} 
 
-decl {\#include "WidgetPDial.h"} {public local
+decl {\#include "WidgetPDial.h"} {public
 } 
 
-decl {\#include <stdio.h>} {public local
+decl {\#include <stdio.h>} {public
 } 
 
-decl {\#include <stdlib.h>} {public local
+decl {\#include <stdlib.h>} {public
 } 
 
-decl {\#include "../globals.h"} {public local
+decl {\#include "../globals.h"} {public
 } 
 
-decl {\#include <FL/Fl_Group.H>} {public local
+decl {\#include <FL/Fl_Group.H>} {public
 } 
 
-decl {\#include "../Params/EnvelopeParams.h"} {public local
+decl {\#include "../Params/EnvelopeParams.h"} {public
 } 
 
-decl {\#include <FL/Fl_Box.H>} {public local
+decl {\#include <FL/Fl_Box.H>} {public
 } 
 
-decl {\#include <FL/fl_draw.H>} {public local
+decl {\#include <FL/fl_draw.H>} {public
 } 
 
-decl {\#include <FL/fl_ask.H>} {public local
+decl {\#include <FL/fl_ask.H>} {public
 } 
 
-decl {\#include "PresetsUI.h"} {public local
+decl {\#include "PresetsUI.h"} {public
 } 
 
-decl {\#include "common.H"} {public local
+decl {\#include "common.H"} {public
 } 
 
 class EnvelopeFreeEdit {: {public Fl_Box}
@@ -192,15 +190,11 @@ if ((event==FL_DRAG)&&(currentpoint>=0)){
 
 return(1);} {}
   }
-  decl {Fl_Box *pair;} {private local
-  }
-  decl {EnvelopeParams *env;} {private local
-  }
-  decl {int oldx,oldy;} {private local
-  }
-  decl {int currentpoint,cpx,cpdt;} {private local
-  }
-  decl {int lastpoint;} {public local
+  decl {Fl_Box *pair;} {}
+  decl {EnvelopeParams *env;} {}
+  decl {int oldx,oldy;} {}
+  decl {int currentpoint,cpx,cpdt;} {}
+  decl {int lastpoint;} {public
   }
 } 
 
@@ -225,7 +219,7 @@ delete (freemodeeditwindow);} {}
   } {
     Fl_Window freemodeeditwindow {
       label Envelope
-      xywh {948 246 575 180} type Double visible
+      xywh {702 269 575 180} type Double visible
     } {
       Fl_Box freeedit {
         label Envelope
@@ -347,7 +341,7 @@ envfree->redraw();}
   Function {make_ADSR_window()} {open
   } {
     Fl_Window envADSR {open
-      xywh {341 765 205 70} type Double color 50 labelfont 1
+      xywh {344 788 205 70} type Double color 50 labelfont 1
       class Fl_Group visible
     } {
       Fl_Group {} {
@@ -427,7 +421,7 @@ freeedit->redraw();}
   Function {make_ASR_window()} {open
   } {
     Fl_Window envASR {open
-      xywh {645 644 210 70} type Double
+      xywh {648 667 210 70} type Double
       class Fl_Group visible
     } {
       Fl_Group {} {
@@ -501,7 +495,7 @@ freeedit->redraw();}
   Function {make_ADSRfilter_window()} {open
   } {
     Fl_Window envADSRfilter {open selected
-      xywh {624 546 275 70} type Double color 50 labelfont 1
+      xywh {627 569 275 70} type Double color 50 labelfont 1
       class Fl_Group visible
     } {
       Fl_Group {} {
@@ -591,7 +585,7 @@ freeedit->redraw();}
   Function {make_ASRbw_window()} {open
   } {
     Fl_Window envASRbw {open
-      xywh {359 619 210 71} type Double
+      xywh {362 642 210 70} type Double
       code0 {set_module_parameters(o);}
       class Fl_Group visible
     } {
@@ -666,7 +660,7 @@ freeedit->redraw();}
   Function {make_free_window()} {open
   } {
     Fl_Window envfree {open
-      xywh {373 413 205 70} type Double color 50 labelfont 1 resizable
+      xywh {376 436 205 70} type Double color 50 labelfont 1 resizable
       code0 {set_module_parameters(o);}
       class Fl_Group visible
     } {
@@ -869,8 +863,6 @@ envwindow->resize(this->x(),this->y(),this->w(),this->h());
 
 envwindow->show();} {}
   }
-  decl {EnvelopeParams *env;} {private local
-  }
-  decl {Fl_Group *envwindow;} {private local
-  }
+  decl {EnvelopeParams *env;} {}
+  decl {Fl_Group *envwindow;} {}
 } 
diff --git a/src/UI/FilterUI.fl b/src/UI/FilterUI.fl
index 9df04f78..0bae429e 100644
--- a/src/UI/FilterUI.fl
+++ b/src/UI/FilterUI.fl
@@ -1,47 +1,44 @@
 # data file for the Fltk User Interface Designer (fluid)
-version 1.0300 
+version 1.0110 
 header_name {.h} 
 code_name {.cc}
-decl {//Copyright (c) 2002-2005 Nasca Octavian Paul} {private local
-} 
+decl {//Copyright (c) 2002-2005 Nasca Octavian Paul} {} 
 
-decl {//License: GNU GPL version 2 or later} {private local
-} 
+decl {//License: GNU GPL version 2 or later} {} 
 
-decl {\#include "WidgetPDial.h"} {public local
+decl {\#include "WidgetPDial.h"} {public
 } 
 
-decl {\#include <cmath>} {private local
-} 
+decl {\#include <cmath>} {} 
 
-decl {\#include <stdio.h>} {private global
+decl {\#include <stdio.h>} {global
 } 
 
-decl {\#include <stdlib.h>} {private global
+decl {\#include <stdlib.h>} {global
 } 
 
-decl {\#include "../globals.h"} {public local
+decl {\#include "../globals.h"} {public
 } 
 
-decl {\#include <FL/Fl_Group.H>} {public local
+decl {\#include <FL/Fl_Group.H>} {public
 } 
 
-decl {\#include "../Params/FilterParams.h"} {public local
+decl {\#include "../Params/FilterParams.h"} {public
 } 
 
-decl {\#include <FL/Fl_Box.H>} {public local
+decl {\#include <FL/Fl_Box.H>} {public
 } 
 
-decl {\#include <FL/fl_draw.H>} {public local
+decl {\#include <FL/fl_draw.H>} {public
 } 
 
-decl {\#include <FL/fl_ask.H>} {public local
+decl {\#include <FL/fl_ask.H>} {public
 } 
 
-decl {\#include "PresetsUI.h"} {public local
+decl {\#include "PresetsUI.h"} {public
 } 
 
-decl {\#include "common.H"} {public local
+decl {\#include "common.H"} {public
 } 
 
 class FormantFilterGraph {open : {public Fl_Box}
@@ -155,14 +152,10 @@ fl_line_style(FL_SOLID,0);} {}
   Function {~FormantFilterGraph()} {} {
     code {delete [] graphpoints;} {}
   }
-  decl {FilterParams *pars;} {private local
-  }
-  decl {int oldx,oldy;} {private local
-  }
-  decl {int *nvowel,*nformant;} {private local
-  }
-  decl {float *graphpoints;} {private local
-  }
+  decl {FilterParams *pars;} {}
+  decl {int oldx,oldy;} {}
+  decl {int *nvowel,*nformant;} {}
+  decl {float *graphpoints;} {}
 } 
 
 class FilterUI {open : {public Fl_Group,PresetsUI_}
@@ -183,7 +176,7 @@ delete (formantparswindow);} {}
   Function {make_window()} {open
   } {
     Fl_Window filterui {open
-      xywh {495 323 275 70} type Double color 50 labelfont 1
+      xywh {498 346 275 70} type Double color 50 labelfont 1
       class Fl_Group visible
     } {
       Fl_Group filterparamswindow {
@@ -353,7 +346,7 @@ pars->changed=true;} open
   } {
     Fl_Window formantparswindow {
       label {Formant Filter Parameters} selected
-      xywh {515 450 700 205} type Double visible
+      xywh {518 473 700 205} type Double visible
     } {
       Fl_Group {} {
         xywh {485 47 105 113} box THIN_UP_BOX
@@ -639,10 +632,7 @@ formant_q_dial->when(0);
 formant_amp_dial->when(0);
 strchdial->when(0);} {}
   }
-  decl {FilterParams *pars;} {private local
-  }
-  decl {unsigned char *velsnsamp,*velsns;} {private local
-  }
-  decl {int nvowel,nformant,nseqpos;} {private local
-  }
+  decl {FilterParams *pars;} {}
+  decl {unsigned char *velsnsamp,*velsns;} {}
+  decl {int nvowel,nformant,nseqpos;} {}
 } 
diff --git a/src/UI/LFOUI.fl b/src/UI/LFOUI.fl
index 52c8a299..889a3861 100644
--- a/src/UI/LFOUI.fl
+++ b/src/UI/LFOUI.fl
@@ -1,44 +1,42 @@
 # data file for the Fltk User Interface Designer (fluid)
-version 1.0300 
+version 1.0110 
 header_name {.h} 
 code_name {.cc}
-decl {//Copyright (c) 2002-2005 Nasca Octavian Paul} {private local
-} 
+decl {//Copyright (c) 2002-2005 Nasca Octavian Paul} {} 
 
-decl {//License: GNU GPL version 2 or later} {private local
-} 
+decl {//License: GNU GPL version 2 or later} {} 
 
-decl {\#include "WidgetPDial.h"} {public local
+decl {\#include "WidgetPDial.h"} {public
 } 
 
-decl {\#include <stdio.h>} {public local
+decl {\#include <stdio.h>} {public
 } 
 
-decl {\#include <stdlib.h>} {public local
+decl {\#include <stdlib.h>} {public
 } 
 
-decl {\#include "../globals.h"} {public local
+decl {\#include "../globals.h"} {public
 } 
 
-decl {\#include <FL/Fl_Group.H>} {public local
+decl {\#include <FL/Fl_Group.H>} {public
 } 
 
-decl {\#include "../Params/LFOParams.h"} {public local
+decl {\#include "../Params/LFOParams.h"} {public
 } 
 
-decl {\#include <FL/Fl_Box.H>} {public local
+decl {\#include <FL/Fl_Box.H>} {public
 } 
 
-decl {\#include <FL/fl_draw.H>} {public local
+decl {\#include <FL/fl_draw.H>} {public
 } 
 
-decl {\#include <FL/fl_ask.H>} {public local
+decl {\#include <FL/fl_ask.H>} {public
 } 
 
-decl {\#include "PresetsUI.h"} {public local
+decl {\#include "PresetsUI.h"} {public
 } 
 
-decl {\#include "common.H"} {public local
+decl {\#include "common.H"} {public
 } 
 
 class LFOUI {open : {public Fl_Group, PresetsUI_}
@@ -54,7 +52,7 @@ hide();
   Function {make_window()} {open
   } {
     Fl_Window lfoui {open selected
-      xywh {627 328 230 70} type Double color 50 labelfont 1
+      xywh {630 351 230 70} type Double color 50 labelfont 1
       class Fl_Group visible
     } {
       Fl_Group lfoparamswindow {
@@ -179,6 +177,5 @@ lfoui->resize(this->x(),this->y(),this->w(),this->h());
 
 lfoparamswindow->label(this->label());} {}
   }
-  decl {LFOParams *pars;} {private local
-  }
+  decl {LFOParams *pars;} {}
 } 
diff --git a/src/UI/MasterUI.fl b/src/UI/MasterUI.fl
index 29b338ec..57ed0166 100644
--- a/src/UI/MasterUI.fl
+++ b/src/UI/MasterUI.fl
@@ -1,74 +1,72 @@
 # data file for the Fltk User Interface Designer (fluid)
-version 1.0300 
+version 1.0110 
 header_name {.h} 
 code_name {.cc}
-decl {//Copyright (c) 2002-2009 Nasca Octavian Paul} {private local
-} 
+decl {//Copyright (c) 2002-2009 Nasca Octavian Paul} {} 
 
-decl {//License: GNU GPL version 2 or later} {private local
-} 
+decl {//License: GNU GPL version 2 or later} {} 
 
-decl {\#include <stdlib.h>} {public local
+decl {\#include <stdlib.h>} {public
 } 
 
-decl {\#include <stdio.h>} {public local
+decl {\#include <stdio.h>} {public
 } 
 
-decl {\#include <string.h>} {public local
+decl {\#include <string.h>} {public
 } 
 
-decl {\#include "WidgetPDial.h"} {public local
+decl {\#include "WidgetPDial.h"} {public
 } 
 
-decl {\#include "ADnoteUI.h"} {public local
+decl {\#include "ADnoteUI.h"} {public
 } 
 
-decl {\#include "SUBnoteUI.h"} {public local
+decl {\#include "SUBnoteUI.h"} {public
 } 
 
-decl {\#include "EffUI.h"} {public local
+decl {\#include "EffUI.h"} {public
 } 
 
-decl {\#include "VirKeyboard.h"} {public local
+decl {\#include "VirKeyboard.h"} {public
 } 
 
-decl {\#include "ConfigUI.h"} {public local
+decl {\#include "ConfigUI.h"} {public
 } 
 
-decl {\#include "BankUI.h"} {public local
+decl {\#include "BankUI.h"} {public
 } 
 
-decl {\#include "PartUI.h"} {public local
+decl {\#include "PartUI.h"} {public
 } 
 
-decl {\#include "MicrotonalUI.h"} {public local
+decl {\#include "MicrotonalUI.h"} {public
 } 
 
-decl {\#include "PresetsUI.h"} {public local
+decl {\#include "PresetsUI.h"} {public
 } 
 
 decl {\#include "NioUI.h"} {public global
 } 
 
-decl {\#include "../Misc/Master.h"} {public local
+decl {\#include "../Misc/Master.h"} {public
 } 
 
-decl {\#include "../Misc/Part.h"} {public local
+decl {\#include "../Misc/Part.h"} {public
 } 
 
-decl {\#include "../Misc/Util.h"} {public local
+decl {\#include "../Misc/Util.h"} {public
 } 
 
-decl {\#include "common.H"} {public local
+decl {\#include "common.H"} {public
 } 
 
 decl {\#if USE_NSM
 \#include "NSM.H"
 extern NSM_Client *nsm;
-\#endif} {public local
+\#endif} {public
 } 
 
-decl {\#include "../globals.h"} {public local
+decl {\#include "../globals.h"} {public
 } 
 
 class VUMeter {: {public Fl_Box}
@@ -268,14 +266,10 @@ float  tmp=ly*1.0/MIN_DB;
 };
 return(1);} {}
   }
-  decl {Master *master;} {private local
-  }
-  decl {int npart;} {private local
-  }
-  decl {float olddbl,olddbr;} {private local
-  }
-  decl {float oldrmsdbl,oldrmsdbr;} {private local
-  }
+  decl {Master *master;} {}
+  decl {int npart;} {}
+  decl {float olddbl,olddbr;} {}
+  decl {float oldrmsdbl,oldrmsdbr;} {}
 } 
 
 class SysEffSend {open : {public WidgetPDial}
@@ -311,12 +305,9 @@ this->copy_label(tmp);} {}
 
 return(WidgetPDial::handle(event));} {}
   }
-  decl {Master *master;} {private local
-  }
-  decl {int neff1;} {private local
-  }
-  decl {int neff2;} {private local
-  }
+  decl {Master *master;} {}
+  decl {int neff1;} {}
+  decl {int neff2;} {}
 } 
 
 class Panellistitem {open : {public Fl_Group}
@@ -324,7 +315,7 @@ class Panellistitem {open : {public Fl_Group}
   Function {make_window()} {open private
   } {
     Fl_Window panellistitem {open
-      private xywh {605 688 100 260} type Double box NO_BOX
+      private xywh {608 711 100 260} type Double box NO_BOX
       class Fl_Group visible
     } {
       Fl_Group panellistitemgroup {open
@@ -435,12 +426,9 @@ panellistitemgroup->redraw();} {}
     code {panellistitem->hide();
 //delete(panellistitem);} {}
   }
-  decl {int npart;} {private local
-  }
-  decl {Master *master;} {private local
-  }
-  decl {BankUI *bankui;} {private local
-  }
+  decl {int npart;} {}
+  decl {Master *master;} {}
+  decl {BankUI *bankui;} {}
 } 
 
 class MasterUI {open
@@ -463,7 +451,7 @@ if ((
     *exitprogram=1;
 };
 \#endif} open
-      xywh {77 347 390 525} type Double align 80 xclass zynaddsubfx visible
+      xywh {80 370 390 525} type Double xclass zynaddsubfx visible
     } {
       Fl_Menu_Bar mastermenu {
         xywh {-5 0 690 25}
@@ -1141,7 +1129,7 @@ if (fl_choice("Exit and leave the unsaved data?","No","Yes",NULL)) {
 	*exitprogram=1;
 };
 \#endif} open
-      xywh {280 239 600 335} type Double align 80 visible
+      xywh {283 262 600 335} type Double visible
     } {
       Fl_Menu_Bar simplemastermenu {
         xywh {0 0 690 25}
@@ -1882,30 +1870,17 @@ virkeys->midich=master->part[npart]->Prcvchn;
 simplerefresh();
 bankui->hide();} {}
   }
-  decl {Master *master;} {private local
-  }
-  decl {MicrotonalUI *microtonalui;} {private local
-  }
-  decl {BankUI *bankui;} {private local
-  }
-  decl {int ninseff,npart;} {private local
-  }
-  decl {int nsyseff;} {private local
-  }
-  decl {int *exitprogram;} {private local
-  }
-  decl {SysEffSend *syseffsend[NUM_SYS_EFX][NUM_SYS_EFX];} {private local
-  }
-  decl {VirKeyboard *virkeyboard;} {private local
-  }
-  decl {ConfigUI *configui;} {private local
-  }
-  decl {int swapefftype;} {private local
-  }
-  decl {char masterwindowlabel[100];} {private local
-  }
-  decl {Panellistitem *panellistitem[NUM_MIDI_PARTS];} {private local
-  }
-  decl {NioUI nioui;} {private local
-  }
+  decl {Master *master;} {}
+  decl {MicrotonalUI *microtonalui;} {}
+  decl {BankUI *bankui;} {}
+  decl {int ninseff,npart;} {}
+  decl {int nsyseff;} {}
+  decl {int *exitprogram;} {}
+  decl {SysEffSend *syseffsend[NUM_SYS_EFX][NUM_SYS_EFX];} {}
+  decl {VirKeyboard *virkeyboard;} {}
+  decl {ConfigUI *configui;} {}
+  decl {int swapefftype;} {}
+  decl {char masterwindowlabel[100];} {}
+  decl {Panellistitem *panellistitem[NUM_MIDI_PARTS];} {}
+  decl {NioUI nioui;} {}
 } 
diff --git a/src/UI/OscilGenUI.fl b/src/UI/OscilGenUI.fl
index 2aa88c25..c616db77 100644
--- a/src/UI/OscilGenUI.fl
+++ b/src/UI/OscilGenUI.fl
@@ -1,62 +1,56 @@
 # data file for the Fltk User Interface Designer (fluid)
-version 1.0300 
+version 1.0110 
 header_name {.h} 
 code_name {.cc}
-decl {//Copyright (c) 2002-2005 Nasca Octavian Paul} {private local
-} 
+decl {//Copyright (c) 2002-2005 Nasca Octavian Paul} {} 
 
-decl {//License: GNU GPL version 2 or later} {private local
-} 
+decl {//License: GNU GPL version 2 or later} {} 
 
-decl {\#include "../Synth/OscilGen.h"} {public local
+decl {\#include "../Synth/OscilGen.h"} {public
 } 
 
-decl {\#include "../Misc/Util.h"} {public local
+decl {\#include "../Misc/Util.h"} {public
 } 
 
-decl {\#include "../Misc/Master.h"} {public local
+decl {\#include "../Misc/Master.h"} {public
 } 
 
-decl {\#include "ResonanceUI.h"} {public local
+decl {\#include "ResonanceUI.h"} {public
 } 
 
-decl {\#include <FL/Fl_Box.H>} {public local
+decl {\#include <FL/Fl_Box.H>} {public
 } 
 
-decl {\#include <FL/Fl_Group.H>} {public local
+decl {\#include <FL/Fl_Group.H>} {public
 } 
 
-decl {\#include <FL/Fl_Slider.H>} {public local
+decl {\#include <FL/Fl_Slider.H>} {public
 } 
 
-decl {\#include <math.h>} {private local
-} 
+decl {\#include <math.h>} {} 
 
-decl {\#include <stdio.h>} {private local
-} 
+decl {\#include <stdio.h>} {} 
 
-decl {\#include <stdlib.h>} {private local
-} 
+decl {\#include <stdlib.h>} {} 
 
-decl {\#include <string.h>} {private local
-} 
+decl {\#include <string.h>} {} 
 
-decl {\#include "WidgetPDial.h"} {public local
+decl {\#include "WidgetPDial.h"} {public
 } 
 
-decl {\#include "EnvelopeUI.h"} {public local
+decl {\#include "EnvelopeUI.h"} {public
 } 
 
-decl {\#include "LFOUI.h"} {public local
+decl {\#include "LFOUI.h"} {public
 } 
 
-decl {\#include "FilterUI.h"} {public local
+decl {\#include "FilterUI.h"} {public
 } 
 
-decl {\#include "PresetsUI.h"} {public local
+decl {\#include "PresetsUI.h"} {public
 } 
 
-decl {\#include <FL/fl_draw.H>} {public local
+decl {\#include <FL/fl_draw.H>} {public
 } 
 
 class OscilSpectrum {: {public Fl_Box}
@@ -130,12 +124,9 @@ for (i=0;i<n;i++){
 }
 delete [] spc;} {}
   }
-  decl {OscilGen *oscil;} {private local
-  }
-  decl {int oscbase;} {private local
-  }
-  decl {Master *master;} {private local
-  }
+  decl {OscilGen *oscil;} {}
+  decl {int oscbase;} {}
+  decl {Master *master;} {}
 } 
 
 class PSlider {: {public Fl_Slider}
@@ -251,14 +242,11 @@ fl_end_line();
 
 fl_line_style(FL_SOLID,0);} {}
   }
-  decl {OscilGen *oscil;} {private local
-  }
-  decl {int oscbase;} {private local
-  }
-  decl {int phase;} {public local
-  }
-  decl {Master *master;} {private local
+  decl {OscilGen *oscil;} {}
+  decl {int oscbase;} {}
+  decl {int phase;} {public
   }
+  decl {Master *master;} {}
 } 
 
 class Oscilharmonic {: {public Fl_Group}
@@ -266,7 +254,7 @@ class Oscilharmonic {: {public Fl_Group}
   Function {make_window()} {open private
   } {
     Fl_Window harmonic {open
-      private xywh {335 236 100 225} type Double box NO_BOX
+      private xywh {338 259 100 225} type Double box NO_BOX
       class Fl_Group visible
     } {
       Fl_Slider mag {
@@ -366,16 +354,11 @@ if (oscil->Phmag[n]==64) mag->selection_color(0);
     code {harmonic->hide();
 //delete(harmonic);} {}
   }
-  decl {OscilGen *oscil;} {private local
-  }
-  decl {Fl_Group *display;} {private local
-  }
-  decl {int n;} {private local
-  }
-  decl {Fl_Widget *oldosc,*cbwidget,*applybutton;} {private local
-  }
-  decl {Master *master;} {private local
-  }
+  decl {OscilGen *oscil;} {}
+  decl {Fl_Group *display;} {}
+  decl {int n;} {}
+  decl {Fl_Widget *oldosc,*cbwidget,*applybutton;} {}
+  decl {Master *master;} {}
 } 
 
 class OscilEditor {open : {public PresetsUI_}
@@ -384,7 +367,7 @@ class OscilEditor {open : {public PresetsUI_}
   } {
     Fl_Window osceditUI {
       label {ADsynth Oscillator Editor} open
-      xywh {544 170 735 595} type Double
+      xywh {542 193 735 595} type Double
       code0 {if (oscil->ADvsPAD) o->label("PADsynth Harmonic Content Editor");} visible
     } {
       Fl_Button applybutton {
@@ -1160,12 +1143,8 @@ if (cbwidget!=NULL) {
       applybutton->redraw();
 };} {}
   }
-  decl {OscilGen *oscil;} {private local
-  }
-  decl {Fl_Widget *oldosc,*cbwidget,*cbapplywidget;} {private local
-  }
-  decl {Oscilharmonic *h[MAX_AD_HARMONICS];} {private local
-  }
-  decl {Master *master;} {private local
-  }
+  decl {OscilGen *oscil;} {}
+  decl {Fl_Widget *oldosc,*cbwidget,*cbapplywidget;} {}
+  decl {Oscilharmonic *h[MAX_AD_HARMONICS];} {}
+  decl {Master *master;} {}
 } 
diff --git a/src/UI/PADnoteUI.fl b/src/UI/PADnoteUI.fl
index aebc11c3..70d26c0f 100644
--- a/src/UI/PADnoteUI.fl
+++ b/src/UI/PADnoteUI.fl
@@ -1,56 +1,52 @@
 # data file for the Fltk User Interface Designer (fluid)
-version 1.0300 
+version 1.0110 
 header_name {.h} 
 code_name {.cc}
-decl {\#include "../Params/PADnoteParameters.h"} {public local
+decl {\#include "../Params/PADnoteParameters.h"} {public
 } 
 
-decl {\#include "../Misc/Util.h"} {public local
+decl {\#include "../Misc/Util.h"} {public
 } 
 
-decl {\#include "../Misc/Master.h"} {public local
+decl {\#include "../Misc/Master.h"} {public
 } 
 
-decl {\#include "ResonanceUI.h"} {public local
+decl {\#include "ResonanceUI.h"} {public
 } 
 
-decl {\#include <FL/Fl_Box.H>} {public local
+decl {\#include <FL/Fl_Box.H>} {public
 } 
 
-decl {\#include <FL/Fl_Group.H>} {public local
+decl {\#include <FL/Fl_Group.H>} {public
 } 
 
-decl {\#include <FL/Fl_File_Chooser.H>} {public local
+decl {\#include <FL/Fl_File_Chooser.H>} {public
 } 
 
-decl {\#include <math.h>} {private local
-} 
+decl {\#include <math.h>} {} 
 
-decl {\#include <stdio.h>} {private local
-} 
+decl {\#include <stdio.h>} {} 
 
-decl {\#include <stdlib.h>} {private local
-} 
+decl {\#include <stdlib.h>} {} 
 
-decl {\#include <string.h>} {private local
-} 
+decl {\#include <string.h>} {} 
 
-decl {\#include "WidgetPDial.h"} {public local
+decl {\#include "WidgetPDial.h"} {public
 } 
 
-decl {\#include "EnvelopeUI.h"} {public local
+decl {\#include "EnvelopeUI.h"} {public
 } 
 
-decl {\#include "LFOUI.h"} {public local
+decl {\#include "LFOUI.h"} {public
 } 
 
-decl {\#include "FilterUI.h"} {public local
+decl {\#include "FilterUI.h"} {public
 } 
 
-decl {\#include "OscilGenUI.h"} {public local
+decl {\#include "OscilGenUI.h"} {public
 } 
 
-decl {\#include "PresetsUI.h"} {public local
+decl {\#include "PresetsUI.h"} {public
 } 
 
 class PADnoteHarmonicProfile {: {public Fl_Box}
@@ -136,9 +132,8 @@ fl_line(ox+lx/2+rbw,oy,ox+lx/2+rbw,oy+ly-1);
 
 fl_line_style(0);} {}
   }
-  decl {Master *master;} {private local
-  }
-  decl {PADnoteParameters *pars;} {public local
+  decl {Master *master;} {}
+  decl {PADnoteParameters *pars;} {public
   }
 } 
 
@@ -232,9 +227,8 @@ for (int i=0;i<lx;i++){
       
 };} {}
   }
-  decl {Master *master;} {private local
-  }
-  decl {PADnoteParameters *pars;} {public local
+  decl {Master *master;} {}
+  decl {PADnoteParameters *pars;} {public
   }
 } 
 
@@ -252,7 +246,7 @@ make_window();} {}
   } {
     Fl_Window padnotewindow {
       label {PAD synth Parameters} open
-      xywh {291 369 535 435} type Double visible
+      xywh {294 392 535 435} type Double visible
     } {
       Fl_Tabs {} {
         callback {if (o->value()!=harmonicstructuregroup) applybutton->hide();
@@ -1113,14 +1107,14 @@ delete(resui);
 padnotewindow->hide();
 delete(padnotewindow);} {}
   }
-  decl {PADnoteParameters *pars;} {public local
+  decl {PADnoteParameters *pars;} {public
   }
-  decl {Master *master;} {public local
+  decl {Master *master;} {public
   }
-  decl {OscilEditor *oscui;} {public local
+  decl {OscilEditor *oscui;} {public
   }
-  decl {Oscilloscope *osc;} {public local
+  decl {Oscilloscope *osc;} {public
   }
-  decl {ResonanceUI *resui;} {public local
+  decl {ResonanceUI *resui;} {public
   }
 } 
diff --git a/src/UI/PartUI.fl b/src/UI/PartUI.fl
index 6066dc36..8e591ac4 100644
--- a/src/UI/PartUI.fl
+++ b/src/UI/PartUI.fl
@@ -1,47 +1,45 @@
 # data file for the Fltk User Interface Designer (fluid)
-version 1.0300 
+version 1.0110 
 header_name {.h} 
 code_name {.cc}
-decl {//Copyright (c) 2002-2005 Nasca Octavian Paul} {private local
-} 
+decl {//Copyright (c) 2002-2005 Nasca Octavian Paul} {} 
 
-decl {//License: GNU GPL version 2 or later} {private local
-} 
+decl {//License: GNU GPL version 2 or later} {} 
 
-decl {\#include <stdlib.h>} {public local
+decl {\#include <stdlib.h>} {public
 } 
 
-decl {\#include <stdio.h>} {public local
+decl {\#include <stdio.h>} {public
 } 
 
-decl {\#include <string.h>} {public local
+decl {\#include <string.h>} {public
 } 
 
-decl {\#include "WidgetPDial.h"} {public local
+decl {\#include "WidgetPDial.h"} {public
 } 
 
-decl {\#include "EffUI.h"} {public local
+decl {\#include "EffUI.h"} {public
 } 
 
-decl {\#include "BankUI.h"} {public local
+decl {\#include "BankUI.h"} {public
 } 
 
-decl {\#include "ADnoteUI.h"} {public local
+decl {\#include "ADnoteUI.h"} {public
 } 
 
-decl {\#include "SUBnoteUI.h"} {public local
+decl {\#include "SUBnoteUI.h"} {public
 } 
 
-decl {\#include "PADnoteUI.h"} {public local
+decl {\#include "PADnoteUI.h"} {public
 } 
 
-decl {\#include "../Misc/Config.h"} {public local
+decl {\#include "../Misc/Config.h"} {public
 } 
 
-decl {\#include "../Misc/Master.h"} {public local
+decl {\#include "../Misc/Master.h"} {public
 } 
 
-decl {\#include "../Misc/Part.h"} {public local
+decl {\#include "../Misc/Part.h"} {public
 } 
 
 class PartSysEffSend {open : {public Fl_Group}
@@ -49,7 +47,7 @@ class PartSysEffSend {open : {public Fl_Group}
   Function {make_window()} {open private
   } {
     Fl_Window syseffsend {
-      private xywh {586 106 100 100} type Double box NO_BOX
+      private xywh {589 129 100 100} type Double box NO_BOX
       class Fl_Group visible
     } {
       Fl_Dial {} {
@@ -80,12 +78,9 @@ end();} {}
     code {syseffsend->hide();
 //delete(syseffsend);} {}
   }
-  decl {Master *master;} {private local
-  }
-  decl {int neff;} {private local
-  }
-  decl {int npart;} {private local
-  }
+  decl {Master *master;} {}
+  decl {int neff;} {}
+  decl {int npart;} {}
 } 
 
 class PartKitItem {open : {public Fl_Group}
@@ -93,7 +88,7 @@ class PartKitItem {open : {public Fl_Group}
   Function {make_window()} {open private
   } {
     Fl_Window partkititem {
-      private xywh {470 383 670 100} type Double box NO_BOX
+      private xywh {473 406 670 100} type Double box NO_BOX
       class Fl_Group visible
     } {
       Fl_Group partkititemgroup {
@@ -267,16 +262,11 @@ end();} {}
     code {partkititem->hide();
 //delete(partkititem);} {}
   }
-  decl {Part *part;} {private local
-  }
-  decl {int n;} {private local
-  }
-  decl {Master *master;} {private local
-  }
-  decl {char label[10];} {private local
-  }
-  decl {class PartUI *partui;} {private local
-  }
+  decl {Part *part;} {}
+  decl {int n;} {}
+  decl {Master *master;} {}
+  decl {char label[10];} {}
+  decl {class PartUI *partui;} {}
 } 
 
 class PartUI {open : {public Fl_Group}
@@ -284,7 +274,7 @@ class PartUI {open : {public Fl_Group}
   Function {make_window()} {open private
   } {
     Fl_Window partgroup {open
-      private xywh {685 241 385 180} type Double box NO_BOX
+      private xywh {688 264 385 180} type Double box NO_BOX
       class Fl_Group visible
     } {
       Fl_Group partgroupui {open
@@ -462,7 +452,7 @@ if (part->Penabled==0) partgroupui->deactivate();
     }
     Fl_Window ctlwindow {
       label Controllers open
-      private xywh {779 238 500 130} type Double box NO_BOX visible
+      private xywh {777 261 500 130} type Double box NO_BOX visible
     } {
       Fl_Check_Button {} {
         label Expr
@@ -645,7 +635,7 @@ else {propta->deactivate();proptb->deactivate();}}
     }
     Fl_Window partfx {
       label {Part's Insert Effects} selected
-      private xywh {551 637 390 145} type Double box NO_BOX visible
+      private xywh {554 660 390 145} type Double box NO_BOX visible
     } {
       Fl_Counter inseffnocounter {
         label {FX No.}
@@ -766,7 +756,7 @@ pthread_mutex_unlock(&master->mutex);}
     }
     Fl_Window instrumentkitlist {
       label {Instrument Kit} open
-      xywh {583 543 670 370} type Double box NO_BOX visible
+      xywh {586 566 670 370} type Double box NO_BOX visible
     } {
       Fl_Button {} {
         label {Close Window}
@@ -847,7 +837,7 @@ if (part->Pkitmode==0) {
     }
     Fl_Window instrumenteditwindow {
       label {Instrument Edit} open
-      xywh {244 602 395 360} type Double box NO_BOX visible
+      xywh {247 621 395 360} type Double box NO_BOX visible
     } {
       Fl_Group {} {
         xywh {0 220 395 110} box UP_FRAME
@@ -1116,26 +1106,15 @@ delete(instrumentkitlist);
 instrumenteditwindow->hide();
 delete(instrumenteditwindow);} {}
   }
-  decl {Part *part;} {private local
-  }
-  decl {Master *master;} {private local
-  }
-  decl {BankUI *bankui;} {private local
-  }
-  decl {ADnoteUI *adnoteui;} {private local
-  }
-  decl {SUBnoteUI *subnoteui;} {private local
-  }
-  decl {PADnoteUI *padnoteui;} {private local
-  }
-  decl {PartSysEffSend *psyef[NUM_SYS_EFX];} {private local
-  }
-  decl {int npart;} {private local
-  }
-  decl {int ninseff;} {private local
-  }
-  decl {int lastkititem;} {private local
-  }
-  decl {PartKitItem *partkititem[NUM_KIT_ITEMS];} {private local
-  }
+  decl {Part *part;} {}
+  decl {Master *master;} {}
+  decl {BankUI *bankui;} {}
+  decl {ADnoteUI *adnoteui;} {}
+  decl {SUBnoteUI *subnoteui;} {}
+  decl {PADnoteUI *padnoteui;} {}
+  decl {PartSysEffSend *psyef[NUM_SYS_EFX];} {}
+  decl {int npart;} {}
+  decl {int ninseff;} {}
+  decl {int lastkititem;} {}
+  decl {PartKitItem *partkititem[NUM_KIT_ITEMS];} {}
 } 
diff --git a/src/UI/ResonanceUI.fl b/src/UI/ResonanceUI.fl
index b112be8c..12e591ab 100644
--- a/src/UI/ResonanceUI.fl
+++ b/src/UI/ResonanceUI.fl
@@ -1,41 +1,35 @@
 # data file for the Fltk User Interface Designer (fluid)
-version 1.0300 
+version 1.0110 
 header_name {.h} 
 code_name {.cc}
-decl {//Copyright (c) 2002-2005 Nasca Octavian Paul} {private local
-} 
+decl {//Copyright (c) 2002-2005 Nasca Octavian Paul} {} 
 
-decl {//License: GNU GPL version 2 or later} {private local
-} 
+decl {//License: GNU GPL version 2 or later} {} 
 
-decl {\#include <FL/Fl_Box.H>} {public local
+decl {\#include <FL/Fl_Box.H>} {public
 } 
 
-decl {\#include <FL/fl_draw.H>} {public local
+decl {\#include <FL/fl_draw.H>} {public
 } 
 
-decl {\#include <FL/Fl_Value_Output.H>} {public local
+decl {\#include <FL/Fl_Value_Output.H>} {public
 } 
 
-decl {\#include <math.h>} {private local
-} 
+decl {\#include <math.h>} {} 
 
-decl {\#include <stdio.h>} {private local
-} 
+decl {\#include <stdio.h>} {} 
 
-decl {\#include <stdlib.h>} {private local
-} 
+decl {\#include <stdlib.h>} {} 
 
-decl {\#include <string.h>} {private local
-} 
+decl {\#include <string.h>} {} 
 
-decl {\#include "../Synth/Resonance.h"} {public local
+decl {\#include "../Synth/Resonance.h"} {public
 } 
 
-decl {\#include "WidgetPDial.h"} {public local
+decl {\#include "WidgetPDial.h"} {public
 } 
 
-decl {\#include "PresetsUI.h"} {public local
+decl {\#include "PresetsUI.h"} {public
 } 
 
 class ResonanceGraph {open : {public Fl_Box}
@@ -187,18 +181,13 @@ return(1);} {}
     code {this->cbwidget=cbwidget;
 this->applybutton=applybutton;} {}
   }
-  decl {Fl_Value_Output *khzvalue;} {private local
-  }
-  decl {Fl_Value_Output *dbvalue;} {private local
-  }
-  decl {Resonance *respar;} {private local
-  }
-  decl {int oldx,oldy;} {private local
-  }
-  decl {float khzval;} {public local
-  }
-  decl {Fl_Widget *cbwidget,*applybutton;} {private local
+  decl {Fl_Value_Output *khzvalue;} {}
+  decl {Fl_Value_Output *dbvalue;} {}
+  decl {Resonance *respar;} {}
+  decl {int oldx,oldy;} {}
+  decl {float khzval;} {public
   }
+  decl {Fl_Widget *cbwidget,*applybutton;} {}
 } 
 
 class ResonanceUI {open : PresetsUI_
@@ -402,10 +391,8 @@ p1st->value(respar->Pprotectthefundamental);
 
 rg->redraw();} {}
   }
-  decl {Resonance *respar;} {public local
-  }
-  decl {ResonanceGraph *rg;} {private local
-  }
-  decl {Fl_Widget *cbwidget,*cbapplywidget;} {private local
+  decl {Resonance *respar;} {public
   }
+  decl {ResonanceGraph *rg;} {}
+  decl {Fl_Widget *cbwidget,*cbapplywidget;} {}
 } 
diff --git a/src/UI/SUBnoteUI.fl b/src/UI/SUBnoteUI.fl
index 4f9ab6bd..245414de 100644
--- a/src/UI/SUBnoteUI.fl
+++ b/src/UI/SUBnoteUI.fl
@@ -1,41 +1,39 @@
 # data file for the Fltk User Interface Designer (fluid)
-version 1.0300 
+version 1.0110 
 header_name {.h} 
 code_name {.cc}
-decl {//Copyright (c) 2002-2005 Nasca Octavian Paul} {private local
-} 
+decl {//Copyright (c) 2002-2005 Nasca Octavian Paul} {} 
 
-decl {//License: GNU GPL version 2 or later} {private local
-} 
+decl {//License: GNU GPL version 2 or later} {} 
 
-decl {\#include <stdlib.h>} {public local
+decl {\#include <stdlib.h>} {public
 } 
 
-decl {\#include <stdio.h>} {public local
+decl {\#include <stdio.h>} {public
 } 
 
-decl {\#include <string.h>} {public local
+decl {\#include <string.h>} {public
 } 
 
-decl {\#include "../globals.h"} {public local
+decl {\#include "../globals.h"} {public
 } 
 
-decl {\#include "WidgetPDial.h"} {public local
+decl {\#include "WidgetPDial.h"} {public
 } 
 
-decl {\#include "EnvelopeUI.h"} {public local
+decl {\#include "EnvelopeUI.h"} {public
 } 
 
-decl {\#include "FilterUI.h"} {public local
+decl {\#include "FilterUI.h"} {public
 } 
 
-decl {\#include "../Misc/Util.h"} {public local
+decl {\#include "../Misc/Util.h"} {public
 } 
 
-decl {\#include "../Params/SUBnoteParameters.h"} {public local
+decl {\#include "../Params/SUBnoteParameters.h"} {public
 } 
 
-decl {\#include "PresetsUI.h"} {public local
+decl {\#include "PresetsUI.h"} {public
 } 
 
 class SUBnoteharmonic {: {public Fl_Group}
@@ -101,10 +99,8 @@ bw->value(127-pars->Phrelbw[n]);} {}
 hide();
 //delete(harmonic);} {}
   }
-  decl {SUBnoteParameters *pars;} {private local
-  }
-  decl {int n;} {private local
-  }
+  decl {SUBnoteParameters *pars;} {}
+  decl {int n;} {}
 } 
 
 class SUBnoteUI {open : {public PresetsUI_}
@@ -113,7 +109,7 @@ class SUBnoteUI {open : {public PresetsUI_}
   } {
     Fl_Window SUBparameters {
       label {SUBsynth Parameters} open
-      xywh {544 466 735 390} type Double visible
+      xywh {542 489 735 390} type Double visible
     } {
       Fl_Scroll {} {
         label scroll open
@@ -449,8 +445,6 @@ make_window();} {}
 SUBparameters->hide();
 delete(SUBparameters);} {}
   }
-  decl {SUBnoteParameters *pars;} {private local
-  }
-  decl {SUBnoteharmonic *h[MAX_SUB_HARMONICS];} {private local
-  }
+  decl {SUBnoteParameters *pars;} {}
+  decl {SUBnoteharmonic *h[MAX_SUB_HARMONICS];} {}
 } 
diff --git a/src/UI/VirKeyboard.fl b/src/UI/VirKeyboard.fl
index 596dcb07..c07a74ea 100644
--- a/src/UI/VirKeyboard.fl
+++ b/src/UI/VirKeyboard.fl
@@ -1,77 +1,63 @@
 # data file for the Fltk User Interface Designer (fluid)
-version 1.0300 
+version 1.0110 
 header_name {.h} 
 code_name {.cc}
-decl {//Copyright (c) 2002-2005 Nasca Octavian Paul} {private local
-} 
+decl {//Copyright (c) 2002-2005 Nasca Octavian Paul} {} 
 
-decl {//License: GNU GPL version 2 or later} {private local
-} 
+decl {//License: GNU GPL version 2 or later} {} 
 
-decl {\#include <stdlib.h>} {public local
+decl {\#include <stdlib.h>} {public
 } 
 
-decl {\#include <FL/fl_draw.H>} {public local
+decl {\#include <FL/fl_draw.H>} {public
 } 
 
-decl {\#include <FL/Fl_Box.H>} {public local
+decl {\#include <FL/Fl_Box.H>} {public
 } 
 
-decl {\#include "../globals.h"} {public local
+decl {\#include "../globals.h"} {public
 } 
 
-decl {\#include "../Misc/Master.h"} {public local
+decl {\#include "../Misc/Master.h"} {public
 } 
 
-decl {\#include "../Misc/Util.h"} {public local
+decl {\#include "../Misc/Util.h"} {public
 } 
 
-decl {\#include "WidgetPDial.h"} {public local
+decl {\#include "WidgetPDial.h"} {public
 } 
 
-decl {\#include "common.H"} {public local
+decl {\#include "common.H"} {public
 } 
 
 decl {\#ifdef NTK_GUI
      \#include "FL/Fl_Shared_Image.H"
-     \#endif} {public local
+     \#endif} {public
 } 
 
-decl {const int keyspos[12]={0,-1,1,-2,2,3,-4,4,-5,5,-6,6};} {private local
-} 
+decl {const int keyspos[12]={0,-1,1,-2,2,3,-4,4,-5,5,-6,6};} {} 
 
-decl {const int keysoct1qwerty[]={'q','2','w','3','e','r','5','t','6','y','7','u','i','9','o','0','p','[','=',']','\\\\',FL_Enter,0};} {private local
-} 
+decl {const int keysoct1qwerty[]={'q','2','w','3','e','r','5','t','6','y','7','u','i','9','o','0','p','[','=',']','\\\\',FL_Enter,0};} {} 
 
-decl {const int keysoct2qwerty[]={'z','s','x','d','c','v','g','b','h','n','j','m',',','l','.',';','/',0};} {private local
-} 
+decl {const int keysoct2qwerty[]={'z','s','x','d','c','v','g','b','h','n','j','m',',','l','.',';','/',0};} {} 
 
-decl {const int keysoct1dw[]={'\\'','2',',','3','.','p','5','y','6','f','7','g','c','9','r','0','l','/',']','=','\\\\',FL_Enter,0};} {private local
-} 
+decl {const int keysoct1dw[]={'\\'','2',',','3','.','p','5','y','6','f','7','g','c','9','r','0','l','/',']','=','\\\\',FL_Enter,0};} {} 
 
-decl {const int keysoct2dw[]={';','o','q','e','j','k','i','x','d','b','h','m','w','n','v','s','z',0};} {private local
-} 
+decl {const int keysoct2dw[]={';','o','q','e','j','k','i','x','d','b','h','m','w','n','v','s','z',0};} {} 
 
-decl {const int keysoct1qwertz[]={'q','2','w','3','e','r','5','t','6','z','7','u','i','9','o','0','p',252,'\\'','+','\\\\',FL_Enter,0};} {private local
-} 
+decl {const int keysoct1qwertz[]={'q','2','w','3','e','r','5','t','6','z','7','u','i','9','o','0','p',252,'\\'','+','\\\\',FL_Enter,0};} {} 
 
-decl {const int keysoct2qwertz[]={'y','s','x','d','c','v','g','b','h','n','j','m',',','l','.',246,'-',0};} {private local
-} 
+decl {const int keysoct2qwertz[]={'y','s','x','d','c','v','g','b','h','n','j','m',',','l','.',246,'-',0};} {} 
 
-decl {const int keysoct1az[]={'a',233,'z','\\"','e','r','(','t','-','y',232,'u','i',231,'o',224,'p',65106,'=','$',0};} {private local
-} 
+decl {const int keysoct1az[]={'a',233,'z','\\"','e','r','(','t','-','y',232,'u','i',231,'o',224,'p',65106,'=','$',0};} {} 
 
-decl {const int keysoct2az[]={'w','s','x','d','c','v','g','b','h','n','j',',',';','l',':','m','!',0};} {private local
-} 
+decl {const int keysoct2az[]={'w','s','x','d','c','v','g','b','h','n','j',',',';','l',':','m','!',0};} {} 
 
 class VirKeys {: {public Fl_Box}
 } {
-  decl {static const int N_OCT=6;} {private local
-  }
-  decl {static const int SIZE_WHITE=14;} {private local
-  }
-  decl {static const int SIZE_BLACK=8;} {private local
-  }
+  decl {static const int N_OCT=6;} {}
+  decl {static const int SIZE_WHITE=14;} {}
+  decl {static const int SIZE_BLACK=8;} {}
   Function {VirKeys(int x,int y, int w, int h, const char *label=0):Fl_Box(x,y,w,h,label)} {} {
     code {master=NULL;} {}
   }
@@ -171,8 +157,7 @@ for (i=0;i<N_OCT*12;i++){
               SIZE_BLACK-3,ly*3/5-5);
    }
 }
-\#endif
-} {}
+\#endif} {}
   }
   Function {handle(int event)} {return_type int
   } {
@@ -295,17 +280,15 @@ pthread_mutex_unlock(&master->mutex);} {}
   Function {relaseallkeys(int type)} {} {
     code {for (int i=0;i<N_OCT*12;i++) relasekey(i,type);} {}
   }
-  decl {Master *master;} {private local
+  decl {Master *master;} {}
+  decl {int pressed[N_OCT*12+1];} {}
+  decl {unsigned char midich;} {public
   }
-  decl {int pressed[N_OCT*12+1];} {private local
+  decl {unsigned char midivel;} {public
   }
-  decl {unsigned char midich;} {public local
+  decl {char midioct,keyoct1,keyoct2;} {public
   }
-  decl {unsigned char midivel;} {public local
-  }
-  decl {char midioct,keyoct1,keyoct2;} {public local
-  }
-  decl {unsigned char rndvelocity;} {public local
+  decl {unsigned char rndvelocity;} {public
   }
 } 
 
@@ -317,7 +300,7 @@ class VirKeyboard {open
       label {Virtual Keyboard - ZynAddSubFX}
       callback {relaseallkeys();
 virkeyboardwindow->hide();} open
-      xywh {97 574 650 130} type Double visible
+      xywh {100 597 650 130} type Double visible
     } {
       Fl_Box virkeys {
         label Keyboard
@@ -452,7 +435,7 @@ virkeys->take_focus();}
 master->setController(virkeys->midich,C_pitchwheel,-(int) o->value());
 pthread_mutex_unlock(&master->mutex);
 virkeys->take_focus();}
-        tooltip {Pitch Wheel} xywh {625 10 20 95} box UP_BOX labelsize 8 align 1 when 3 minimum -8192 maximum 8192 step 64
+        tooltip {Pitch Wheel} xywh {625 10 20 95} labelsize 8 align 1 when 3 minimum -8192 maximum 8192 step 64
       }
       Fl_Button {} {
         label R
@@ -492,8 +475,6 @@ make_window();} {}
   Function {relaseallkeys()} {} {
     code {virkeys->relaseallkeys(0);} {}
   }
-  decl {Master *master;} {private local
-  }
-  decl {int midictl;} {private local
-  }
+  decl {Master *master;} {}
+  decl {int midictl;} {}
 } 
-- 
2.47.0

